flink-end-to-end-tests/flink-tpcds-test/tpcds-tool/data_generator.sh

```diff
@@ -130,6 +130,10 @@ elif  [[ "$OS_TYPE" == "linux" ]]; then
         chmod +x  $generator_dir/dsdgen_linux
         cd  $generator_dir
         ./dsdgen_linux -SCALE $scale_factor -FORCE Y -DIR $data_dir
+        echo "[INFO] `date +%H:%M:%S` Convert file encoding of customer.dat to UTF-8 start."
```

Comment:
Would you add a comment to explain that "the data generator may generate files encoded in latin1, which the new csv source cannot read"? 
And looks to me this is a bug of the new csv source and needs to be fixed. After it is fixed, we can remove this workaround.

Reply:
+1, maybe add a "TODO" comments. I 've created an issue for the bug: FLINK-26760

Reply:
I will add a "TODO" comments here.